POC/ColorPicker/ColorPicker/TransparentWindow.xaml.cs

```diff
@@ -35,28 +68,35 @@ private void OnKeyDown(object sender, System.Windows.Input.KeyEventArgs e)
             }
         }
 
-        private void SetWindowSizeToCoverAllScreens()
+        private void OnMouseMove(object sender, System.Windows.Input.MouseEventArgs e)
         {
-            int minX = 0;
-            int maxX = 0;
+            RealignMagnificationBox(e);
+        }
 
-            int minY = 0;
-            int maxY = 0;
+        private void ConfigureUpdateTimer()
+        {
+            _updateTimer.Tick += RefreshMagnificationBox;
+            _updateTimer.Interval = new TimeSpan(1000);
```

Comment:
I think you can use TimeSpan.FromSeconds (or milliseconds), which is easier to understand (don't have to think what the unit is0